### PR TITLE
Add a parser test.

### PR DESCRIPTION
**Pull request by mikenikles**
Adds a basic parser test.

**Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/123

## Changes
- Added a `home.test.js` parser test.

## How to test-drive this PR
- Point the `progressive-web-sdk` dependency to https://github.com/mobify/progressive-web-sdk/pull/123.
- `npm t`

## Todo

- [x] Release the changes in the progressive-web-sdk, update `package.json` to point to the new release and make sure CircleCI tests pass.

### CODE REVIEW

File: app/containers/home/parsers/home.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import test from 'ava'
+import {jquerifyHtmlFile} from 'progressive-web-sdk/dist/test-utils'
```
Comment (marlowpayne) [inline app/containers/home/parsers/home.test.js]: We'll need to be committing our `dist/` directory in the SDK, and publish it on NPM before this line works correctly on CircleCI, right?

Comment (mikenikles) [inline app/containers/home/parsers/home.test.js]: The `dist` directory is already part of the deployed SDK, but it needs the changes from https://github.com/mobify/progressive-web-sdk/pull/123.

In https://github.com/mobify/progressive-web-sdk/pull/124, I changed the way we generate the `dist` directory, but that is independently from fixing CircleCI tests for this PR here.

**Comment by mikenikles** [comment]
@marlowpayne we're back in business :). Any final words before I merge this?

**Comment by marlowpayne** [comment]
Final words: :shipit: 👍

**Pull request merged**